omnibus/config/software/sequel-gem.rb

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
Let's stick with using the pg gem, which we already have available. sequel is currently used only for migration from old 11.early sequel-based migrations, and will hopefully be getting yanked soon.